Directory.Build.props

```diff
@@ -0,0 +1,38 @@
+<Project>
+
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <LangVersion>8.0</LangVersion>
+    <TreatWarningsAsErrors Condition="'$(Configuration)' == 'Release'">true</TreatWarningsAsErrors>
+    <NoWarn>$(NoWarn);CS1591;SA1633;SA1200;SA1201</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>A client library for KMD Logic File Security</Description>
+    <VersionPrefix>0.1.0</VersionPrefix>
```

Comment:
Shouldn't this be a major version? 1.0.0

Reply:
yes. Will update it when I enable nuget publish in yml and will add in `CHANGES.md` as `created nuget pkg`. As of now it is initial version in that document.